Calculate/Calculate/Calculator.cs

```diff
@@ -0,0 +1,72 @@
+namespace Calculate;
+
+public class Calculator
+{
+    public static double Add(int a, int b)
+    {
+        return a + b;
+    }
```

Comment:
```suggestion
    public static double Add(int a, int b) => a + b;
```
for simple methods, you can make them a single line